server/game/cards/11.4-MoD/MeeraReed.js

```diff
@@ -0,0 +1,41 @@
+const DrawCard = require('../../drawcard');
+
+class MeeraReed extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onSacrificed: event => this.isControlledStarkCharacter(event.cardStateWhenSacrificed)
+            },
+            handler: context => {
+                this.game.addMessage('{0} uses {1} to return {1} to shadows', context.player, this);
+                context.player.putIntoShadows(this, false);
+            }
+        });
+
+        this.reaction({
+            when: {
+                onCardOutOfShadows: event => event.card === this
+            },
+            target: {
+                activePromptTitle: 'Select a character',
```

Comment:
Just an FYI: this is the default.

Reply:
Thanks! I actually looked it up while implementing Baratheon cards earlier. I had thought "Select a card" was the default for some reason. Gotta shake off that rust in my brain after not implementing cards for so long. 😄